tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_full.py

```diff
@@ -302,6 +302,37 @@ def dir_size():
     assert dir_size() == 0
 
 
+def test_large_manifests_passed_by_file(wandb_init, monkeypatch, mocker):
+    writer_spy = mocker.spy(
+        wandb.sdk.interface.interface.InterfaceBase,
+        "_write_artifact_manifest_file",
+    )
```

Comment:
note: spying on private methods is a brittle pattern, but I don't have an alternative suggestion at the moment